test/functionalTest/cases/1080_batch_query_context/batch_query_context_error_conditions.test

```diff
@@ -327,21 +310,7 @@ Date: REGEX(.*)
 }
 
 
-10. POST /v2/op/query, with a scope with an unknown field (not 'value' nor 'type') - see error
```

Comment:
This is a consecuence of the changing in processing strategy at parseSubscription(), based in FindMember():

```
Value::ConstMemberIterator iter = valueP->FindMember("type");
if (iter == valueP->MemberEnd())
   {
     return "scope without a type";
   }
```

The bad thing is that we lose the ability to detect not-expected fields. The good thing is that we use one lookup instead two ones (according to http://rapidjson.org/md_doc_tutorial.html#QueryObject), so I think is better  in this way.